.github/workflows/velox.yml

```diff
@@ -0,0 +1,76 @@
+name: Velox Nightly Build and Test
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  container:
+    runs-on: ubuntu-latest
+    container: prestocpp/velox-velox-torcharrow:kpai-20220301
+    steps:
+      - name: Print CPU info
+        run: cat /proc/cpuinfo
+
+      - name: Setup Python environment
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Check out source repository
+        uses: actions/checkout@v2
+        with:
+            submodules: true
+
+      - name: Update submodules
+        run: |
+          git submodule update --init --recursive
+          cd csrc/velox/velox/
+          git submodule update --remote --merge
+
+      - name: Setup conda environment
+        run: | 
+          source /opt/conda/etc/profile.d/conda.sh
+          conda create --name ta_python37 python=3.7 -y
```

Comment:
Also when you run 
```
PYTHON_VERSION=3.7 packaging/build_wheel.sh
```

It will use conda to initialize the Python environment:

https://github.com/facebookresearch/torcharrow/blob/4d21b119bb4e3d7a9db0a9b14d9fec3d3a1b9dcd/packaging/build_wheel.sh#L45-L50

Reply:
I will still need conda to build TorchArrow and run unit tests right ?